src/git.py

```diff
@@ -79,10 +79,10 @@ def _parse_gerrit_patch_push(gerrit_result: str) -> str:
     logging.info('change_id = %s', change_id)
     return change_id
 
-class GerritGit(object):
-    def __init__(self, git_dir: str, cookie_jar_path: str, url: str, project: str, branch: str) -> None:
+
+class DryrunGerritGit(object):
```

Comment:
nit: perhaps rename to `DryRun`

Also nit: perhaps drop `object`, https://google.github.io/styleguide/pyguide.html#39-classes